package TryCatch_;

public class Throwexeptionlogin extends Exception {
	public Throwexeptionlogin() {
		 super("User Not Found");
	 }
}
